src/main/java/frc/robot/RobotContainer.java

```diff
@@ -138,10 +137,10 @@ private void configureButtonBindings()
     {
         _drive.setDefaultCommand(DriveCommands.joystickDrive(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), () -> -_joystick.getZ()));
 
-        _controller.y().whileTrue(NotepathCommands.startFeed(_notepath).andThen(Commands.idle(_notepath)).finallyDo(() -> _notepath.setOff()));
-        _controller.x().whileTrue(NotepathCommands.reverseFeed(_notepath).andThen(Commands.idle(_notepath)).finallyDo(() -> _notepath.setOff()));
-        _controller.a().whileTrue(IntakeCommands.startIntake(_intake).andThen(Commands.idle(_intake)).finallyDo(() -> _intake.setIntakeOff()));
-        _controller.b().whileTrue(IntakeCommands.reverseIntake(_intake).andThen(Commands.idle(_intake)).finallyDo(() -> _intake.setIntakeOff()));
+        _controller.y().onTrue(CompositeCommands.intakePickup(_intake, _notepath).andThen(CompositeCommands.stopIntaking(_intake, _notepath)));
```

Comment:
We don't need to tell it to stop intaking. The composite command already takes care of this

```suggestion
        _controller.y().onTrue(CompositeCommands.intakePickup(_intake, _notepath));
```